internal/pkg/service/common/task/node.go

```diff
@@ -300,21 +300,22 @@ func (n *Node) runTask(logger log.Logger, task Task, cfg Config) (result Result,
 		}
 	}
 
+	// Create context for task finalization, the original context could have timed out.
+	// If release of the lock takes longer than the ttl, lease is expired anyway.
+	ctx, finalizationCancel := context.WithTimeout(context.Background(), time.Duration(n.config.ttlSeconds)*time.Second)
+	defer finalizationCancel()
+
```

Comment:
V telemetrii vidim, ze pocet `Running Tasks` nezmyselne stale narasta:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/77898d4a-9e98-4099-84b4-7f5b8cc35310)

Debugoval som to:
- Vela taskov konci timeoutom, dovodom su ine chyby, riesene inde.
- Problem je, ze ked `ctx` tasku skonci, tak sa neodosle hodnota metriky `running tasks` `-1`.
- Preto potrebujeme vytvorit novy `ctx` na ukonencenie tasku.
- To sa v kode uz robilol, vid nizsie `opCtx`, ale pouzivalo sa to iba na release zamku.
- Tak som to v kode dal vyssie, aby to zahrnalo aj telemetriu.

Reply:
Po vyskusani fixu:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/ba3ad8fa-68e2-4ada-b885-8a8dcea721f3)